guides/release/getting-started/quick-start.md

```diff
@@ -64,6 +65,11 @@ Open [`http://localhost:4200`](http://localhost:4200) in your browser of choice.
 You should see an Ember welcome page and not much else.
 Congratulations! You just created and booted your first Ember app.
 
+If you are having trouble getting this running, other Ember developers would be happy to help!
```

Comment:
I would inline some of the most important information here inside a Zoey Says box.